common.gypi

```diff
@@ -477,6 +477,12 @@
           '-Wl,--export-dynamic',
         ],
       }],
+      ['node_shared_openssl!="true" and node_shared!="true"', { # if node is built as shared lib, the openssl mechanism for keeping itself "dlload"-ed for atexit cleanup should not be disabled
```

Comment:
Can you make sure lines stay <= 80 columns? LGTM apart from that.

Reply:
Splitted the comments into multiple lines, looks a bit "ugly" and less readable to me, to keep the 80 col limit.